pkg/apiserver/debugapi/endpoint_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debugapi
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/joomcode/errorx"
+	. "github.com/pingcap/check"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/model"
+)
+
+func TestT(t *testing.T) {
+	CustomVerboseFlag = true
+	TestingT(t)
+}
+
+var _ = Suite(&testSchemaSuite{})
+
+type testSchemaSuite struct{}
+
+var testTiDBIPParam EndpointAPIParam = EndpointAPIParam{
+	Name:  "tidb_ip",
+	Model: EndpointAPIModelIPPort,
```

Comment:
I think the `IPPort` model can be removed, since every API "instance" needs it, and there is no need to mark each API model to have a IPPort to fill.